app/components/reminder-card.js

```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
```

Comment:
Seems legit but lets put some code in here if we want to use a component for an individual reminder.